webpack.config.js

```diff
@@ -6,6 +6,7 @@ module.exports = {
   entry: {
     index: './src/index.js',
     print: './src/gen_html.js',
+    check: './src/check_complete.js',
```

Comment:
- [ ] I am to run your app on my local server. When I do, I get this error and of course, the server does not open: 👇 
![image](https://user-images.githubusercontent.com/68167810/129957213-5e405e4e-d003-4762-b158-386e9ec7d288.png)Are you also facing the same issue, if not could you ask your partner to confirm that?